apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/claimSearchPaginated.json

```diff
@@ -150,8 +151,8 @@
           "reference": "#patient"
         },
         "billablePeriod": {
-          "start": "1970-07-18T00:00:00+00:00",
-          "end": "1970-07-20T00:00:00+00:00"
+          "start": "1970-07-18",
+          "end": "1970-07-20"
```

Comment:
Confirming this should just be a date and not a datetime?

Reply:
@lsmitchell yes thats right, the claim service dates provided to us by FISS and MCS do not contain any information on the time of when the service was provided.  You can take a look at IDR-HDR-FROM-DOS & IDR-HDR-TO-DOS for MCS claims and STMT-COV-FROM-DATE-CYMD & STMT-COV-TO-DATE-CYMD for FISS claims in the [RDA API data dictionary.](https://docs.google.com/spreadsheets/d/19CGXLO9SGTgaSvACRVuuOatvs4z1NXHzxtI-zt_j8Qc/edit#gid=283461576)